card.js

```diff
@@ -0,0 +1,6 @@
+const cardCloseBtn=document.querySelector('.close-card-btn');
+const cardContainer=document.querySelector('.card__container-close');
+function dismissCard(){
+    cardContainer.style.display="none";
+}
+cardCloseBtn.addEventListener('click',dismissCard);
```

Comment:
General comments for this file
- add space before and after `=`
- add space after comma
- add space before `{` in function definition
- leave line breaks that separate the file into logical blocks (eg. here, we can have line breaks before and after the function definition

Reply:
Thank you for reviewing the PR @manujagobind .
Will surely make the suggested changes.